README.md

```diff
@@ -30,14 +30,16 @@ Alternatively, build and run the plugin manager yourself from source:
 
 ```bash
 mvn clean install 
-java -jar plugin-management-cli/target/jenkins-plugin-manager-*.jar --war /file/path/jenkins.war --plugin-file /file/path/plugins.txt --plugins delivery-pipeline-plugin:1.3.2 deployit-plugin
+java -jar plugin-management-cli/target/jenkins-plugin-manager-*.jar --war /file/path/jenkins.war --plugin-download-directory /your/path/to/plugins/ --plugin-file /file/path/plugins.txt --plugins delivery-pipeline-plugin:1.3.2 deployit-plugin
 ```
 
 If you use a [Jenkins docker image](https://hub.docker.com/r/jenkins/jenkins) the plugin manager can be invoked inside the running container via the bundled `jenkins-plugin-cli` shell script:
 
 ```bash
+docker cp /your/path/to/plugins.txt <container_name>:/tmp/plugins.txt
 docker exec -it <container_name> /bin/bash
-jenkins-plugin-cli --plugin-file /your/path/to/plugins.txt --plugins delivery-pipeline-plugin:1.3.2 deployit-plugin
+jenkins-plugin-cli --plugin-download-directory /your/path/to/plugins/ --plugin-file /tmp/plugins.txt --plugins delivery-pipeline-plugin:1.3.2 deployit-plugin
```

Comment:
The plugin-download-directory argument is not helpful for the docker case.  The default download directory is already the correct location.  The `cp` command is necessary because the default download location contents need to be copied to the runtime location
```suggestion
jenkins-plugin-cli --plugin-file /tmp/plugins.txt --plugins delivery-pipeline-plugin:1.3.2 deployit-plugin
cp -r -p /usr/share/jenkins/ref/plugins/. /var/jenkins_home/plugins/.
```